R/conf_mat_resampled.R

```diff
@@ -36,22 +36,21 @@
 conf_mat_resampled <- function(x, ..., parameters = NULL, tidy = TRUE) {
   rlang::check_dots_empty()
   if (!inherits(x, "tune_results")) {
-    rlang::abort(
-      "The first argument needs to be an object with class 'tune_results'."
-    )
+    cli::cli_abort("The first argument needs to be {.cls tune_results} object,
+                   not {.obj_type_friendly {mtcars}}.")
   }
   if (!any(names(x) == ".predictions")) {
-    rlang::abort(
-      paste0(
-        "The function was not run with the `save_pred = TRUE` option. ",
-        "Please re-run with that option."
-      )
+    cli::cli_abort(
+      "The function was not run with the {.code save_pred = TRUE} option.
```

Comment:
Possibly a thing for a separate issue, but "The function" here seems to refer to the calling env rather than `tune_grid()`/`fit_resamples()`/etc